spec/support/capybara.rb

```diff
@@ -43,26 +43,13 @@ def accept_confirm_dialog
       expect(page).to have_no_selector('button.confirm-btn')
     end
 
-    # Helper to fill in summernote textareas. Only to be used where javascript is enabled.
-    #
-    # The method provides an alternative method to fill up the textarea, which would
-    # otherwise be very slow if capybara's +find+ and +set+ were used.
-    def fill_in_summernote(selector, text)
-      script = <<-JS
-      var editorSelector = '#{selector}';
-      $(editorSelector).summernote('reset');
-      $(editorSelector).summernote('editor.insertText', '#{text}');
-      JS
-      execute_script(script)
-    end
-
-    # Special helper to fill in summernote textarea defined by react.
+    # Special helper to fill in draftjs textarea defined by react.
     #
     # Selector should specify the class of the target +textarea+, and method targets the +div+
     # within. This should change when the internals of the summernote react component is changed.
-    def fill_in_react_summernote(selector, text)
-      react_selector = ' + div.material-summernote > div > div[id^="react-summernote-"]'
-      fill_in_summernote(selector + react_selector, text)
+    def fill_in_draft_js(selector, text)
+      react_selector = ' + div.react-draft > div .public-DraftStyleDefault-block'
```

Comment:
Ref: https://spectrum.chat/testing-library/help/has-anyone-successfully-used-rtl-to-test-change-events-on-a-draftjs-text-box~f1928053-9e53-407a-9be5-70babeb4b692?m=MTU1NzUwMDM0NjAwMg==